examples/route-modal/README.md

```diff
@@ -0,0 +1,19 @@
+# Route modal
+
+Example of using routes to control visibility of modals.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+<!-- TODO: update this link to the path for your example: -->
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/route-modal)
+
+## Example
+
+Describe the example and how it demonstrates solving the problem. Reference any relevant files/dependencies if needed.
+
+## Related Links
+
+Link to documentation or other related examples.
```

Comment:
Could you fill these two bits in?

Reply:
Added a short description and link to React Router v6 Modal example 👍